### PR TITLE
add get license-info-endpoint-to-agent-docs

### PR DESCRIPTION
**Pull request by MaciejSzaflik**
# Deploy preview

Scroll down to the checks section for the link to the deploy preview of your branch.

# Links

Link your Jira ticket.

- [Jira](https://livechatinc.atlassian.net/browse/LC-2270)

# Description

# Review and release

Apply changes and resolve conflicts. Once the described functionality lands on prod, let us know on #platform-docs-releases that your PR is ready for release. We will **merge** and **publish** the changes.

### CODE REVIEW

**Comment by ale-kacperczyk** [comment]
1. Will this be in v3.6 as well? If so, could you update v3.6 docs as well? 
2. Also, I think we should update v3.5 changelog in the following format: 
image

**Comment by MaciejSzaflik** [comment]
> 1. Will this be in v3.6 as well? If so, could you update v3.6 docs as well?
> 
>     2. Also, I think we should update v3.5 changelog in the following format (Get Chat update):
> 
> 
> image

1. as for now it's only available in 3.5
2. added to changelog - but I'm not sure if I should change date in header or not

File: src/pages/messaging/agent-chat-api/changelog/index.mdx [hunk 1]
```diff
@@ -42,6 +42,7 @@ The developer preview version provides a preview of the upcoming changes to the
 
 - The **List Archives** ([Web](/messaging/agent-chat-api/v3.5/#list-archives) & [RTM](/messaging/agent-chat-api/v3.5/rtm-reference/#list-archives)) method has a new filter, `customer_countries`.
 - 🛠️ (2022-12-7): The **Get Chat** ([Web](/messaging/agent-chat-api/v3.5/#get-chat) & [RTM](/messaging/agent-chat-api/v3.5/rtm-reference/#get-chat)) method now validates access to the requested thread.
+- 🛠️ (2024-07-5): The new method **Get License Info** ([Web](/messaging/agent-chat-api/v3.5/#get-license-info)).
```
Comment (ale-kacperczyk) [inline src/pages/messaging/agent-chat-api/changelog/index.mdx:45]:
```suggestion
- 🛠️ (2024-07-05): There's a new method, **Get License Info** ([Web](/messaging/agent-chat-api/v3.5/#get-license-info)).
```

**Comment by ale-kacperczyk** [comment]
> > 1. Will this be in v3.6 as well? If so, could you update v3.6 docs as well?
> >    
> >    2. Also, I think we should update v3.5 changelog in the following format (Get Chat update):
> > 
> > image
> 
> 1. as for now it's only available in 3.5
> 2. added to changelog - but I'm not sure if I should change date in header or not

The date in the headline should stay as is so its ok

**Review by ale-kacperczyk — APPROVED** [review]

**Pull request merged**